src/components/projects/ProjectTasks/TaskCard/index.tsx

```diff
@@ -0,0 +1,147 @@
+'use client';
+
+import { ITasks, Status } from '@/types/database.interface';
+import { Poppins } from 'next/font/google';
+import { DeleteTask, SelectTask } from '../client';
+import { useEffect, useState } from 'react';
+import { createSupbaseClient } from '@/lib/supabase/client';
+
+const PoppinsLight = Poppins({
+	subsets: ['latin-ext'],
+	weight: ['300'],
+});
+
+const PoppinsSemiBold = Poppins({
+	subsets: ['latin-ext'],
+	weight: ['600'],
+});
+
+export function ProjectTask({ task, role }: { task: ITasks; role: string }) {
+	const { id, project_id, title, due_date, completed_date, status } = task;
+
+	const [members, setMembers] = useState<string[]>();
+
+	useEffect(() => {
+		const fetchMembers = async () => {
```

Comment:
For now leave but lets leave comment in Jira informing of this as this will cause ton of API calls as well.

Reply:
roger that